include/exadg/operators/operator_base.cpp

```diff
@@ -242,6 +242,20 @@ OperatorBase<dim, Number, n_components>::set_constrained_values_to_zero(VectorTy
   }
 }
 
+template<int dim, typename Number, int n_components>
+bool
+OperatorBase<dim, Number, n_components>::check_constrained_values_are_zero(
+  VectorType const & vector) const
+{
+  for(unsigned int i = 0; i < constrained_indices.size(); ++i)
+  {
+    if(std::abs(vector.local_element(constrained_indices[i])) > std::numeric_limits<Number>::min())
```

Comment:
I would argue that if `std::abs(vector.local_element(constrained_indices[i]))` in not `0.0` actual there is already an inconsistency in the code.

Reply:
What do you mean? write `== 0.0`? where is the difference?

Reply:
I would also say that we should write `== 0.0` here. There are subnormal numbers between the `min` numeric limit and 0 (so some random content could end up being the same here).

Reply:
`== 0.0` is what we want to assert. So should I read the exceptional case with `std::abs(value) > 0.0`, or `value != 0.0`?

Reply:
!= 0